### ISSUE TITLE
Dev destroy object doesn't seem to work on certain objects

### ISSUE THREAD

**Issue by Aranclanos** [unitystation/unitystation#1776]
# Description
When clicking on top of some objects of the game, the dev destroy option doesnt seem to work on airlocks, scrubbers, vents, tables, wallmounts, windows and maybe more

## Category
BUG

## Steps to Reproduce
Please enter the steps to reproduce the bug or behaviour:
1.Select dev destroy object option
2.Click on the list above
3.Notice they ignore your clicking and do not get deleted

**Comment by chairbender** [comment]
It specifically avoids trying to delete wallmounts and tiles. This could be expanded, but I tried to avoid them because I didn't want to figure out how they sync their state over the network, since they don't have a CNT so they need to be synced in a different way when creating / destroying. Probably not too hard to figure out though.

**Comment by fomalsd** [comment]
airlocks do have CNT though

**Comment by chairbender** [comment]
yeah, but I think the dev deleter is excluding by layer, so doors are one of the layers that are excluded
